scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -1140,7 +1144,9 @@ sealed trait SCollection[T] extends PCollectionWrapper[T] {
    * @group output
    */
   def saveAsTextFile(path: String,
-                     suffix: String = ".txt", numShards: Int = 0): Future[Tap[String]] = {
+                     suffix: String = ".txt", numShards: Int = 0,
```

Comment:
Could you move `numShards: Int = 0` to next line?

Reply:
👍